app/lib/electron/index.js

```diff
@@ -110,6 +110,11 @@ class ElectronRunner {
         log(`Bundling app with electron-${bundlerName}...`)
         log()
 
+        if (cfg.ctx.debug) {
+          console.log(`DEBUG MODE - bundlerConfig:`)
+          console.log(bundlerConfig)
```

Comment:
use log not console.log please

Reply:
I started with just plain `log` (yes, that is a Ren & Stimpy reference) but that outputs:
```
app:electron DEBUG MODE - bundlerConfig: +2ms
app:electron [object Object] +2ms
```

while `console.log` actually displays the JS object's contents. I also don't think the `app:electron` prefix is accurate, because the display is triggered from the `debug` flag, not something in electron or any of its builders.

Reply:
well, personally I think `log` is better than bad, its good - so why not just destructure your object with JSON.parse?

Reply:
Or just use console.log for the bundlerConfig and log for line 113. This seems like it could be valid, and it won't break the universe. Of course, its just my opinion. Razvan will have to sign off on it anyway.